app/api/issue_api.rb

```diff
@@ -35,6 +40,9 @@ def bbox_from_string(string, factory)
     params do
       optional :bbox, type: String, desc: 'Four comma-separated coordinates making up the boundary of interest, e.g. "0.11905,52.20791,0.11907,52.20793"'
       optional :tags, type: Array, desc: 'An array of tags all the issues must have, e.g. ["taga","tagb"]', coerce_with: JSON
+      optional :group, type: String, desc: 'Return only issues from area of group given by it\'s short name'
+      optional :order, type: String, desc: 'Order of returned issues. Current working parameters are: "vote_count", "created_at"'
+      optional :count, type: Integer, desc: 'Limit number of returned issues'
```

Comment:
No quite, `\` is escaping the single quote as the string is in single quotes.
You can do `"'Return only issues from area of group given by it's short name"` where you use double quotes.